content/python/concepts/strings/terms/join/join.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.join()'
+Description: 'Concatenates all items from an iterable into a single string.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.join()` method concatenates all items from an iterable into a single string.
+
+## Syntax
+
+```pseudo
+string.join(iterable)
+```
+
+- The `.join()` method is called on a separator of type string. The separator can be any string, including an empty string, and is used to separate each element of the iterable.
```

Comment:
```suggestion
- The `separator` can be any string, even an empty one, and is placed between each element from the `iterable`.
```

Reply:
This makes more sense! One bullet point for the separator and a second point for the iterable 👍 agreed

Reply:
So change `separator` and `iterable` to just separator and iterable?

Reply:
This is very "nitty-gritty."

Notice how [.replace()](https://www.codecademy.com/resources/docs/python/strings/replace) and [.index()](https://www.codecademy.com/resources/docs/python/strings/index) wrap the argument placeholders in backticks, but for [.partition()](https://www.codecademy.com/resources/docs/python/strings/partition), the keyword is not wrapped in backticks.

A similar issue in [PR 537 file](https://github.com/Codecademy/docs/pull/537/files).

So maybe for these sentence/paragraph list items, we'll do just as you said which is to change `separator` and `iterable` to separator and iterable

Reply:
@yangc95 maybe down the line we can have a consensus on this. I personally like using backticks around these fields only because we don't get to use many emphasis features like italics or bold.

Reply:
@yangc95 @Dusch4593 But to be clear, we are removing the backticks, right?

Reply:
Sorry, I keep going back and forth. Until there's a consensus on this, let's keep the backticks for everything except "string"

Reply:
ok 😂. I'll just commit everything so we can see how it would look. We can always change it again later.

Reply:
🥲 😆 Sounds good